keystore_test.go

```diff
@@ -182,60 +146,43 @@ func TestAliases(t *testing.T) {
 
 	sort.Strings(actualAliases)
 
-	if !reflect.DeepEqual(expectedAliases, actualAliases) {
-		t.Fatal("aliases must be equal")
-	}
+	assert.True(t, reflect.DeepEqual(expectedAliases, actualAliases), "aliases must be equal")
```

Comment:
Please consider using assert.Equal and put expected and actual arguments in a correct order.